docs/src/lib/indexing.md

```diff
@@ -83,6 +84,15 @@ Under construction
 
 ## Broadcasting
 
+The following broadcasting rules apply to `AbstractDataFrame` objects:
+* `AbstractDataFrame` behaves in broadcasting like a two-dimensional collection compatible with matrices.
+* If an `AbstractDataFrame` takes part in broadcasting then a `DataFrame` is always produced as a result.
+  In this case the requested broadcasting operation must produce exactly two dimensional object.
+  An exception is when `AbstractDataFrame` is used only as a source of broadcast assignment into an object
+  of dimensionality higher than two. In such a case this is allowed and a data frame is treated as two-dimensional
```

Comment:
The last sentence seems redundant with what has been said above.

Reply:
ok. removed